### DISCUSSION TITLE
Log in users with low uid's

### DISCUSSION THREAD

**Discussion by mecrider** [ltsp/ltsp#513] [Q&A] [answered]
I am migrating a server from Ubuntu 16.04 and LTSP 5 to Ubuntu 20.04 with current LTSP and Gnome Shell. Some of the users date back to before 16.04, and have UIDs between 500 and 1000. I can log in to the server console with them, and I can ssh to the server using them, but if I log in at an LTSP workstation I get the message "Sorry, that didn't work. Please try again." All users with UIDs above 1000 work fine. Is there a config file somewhere that I can change the minimum UID to login? Previous servers I set up used LDAP for authentication, and I had to update 5 files in /etc/pam.d to a minimum of 500, but this is my first server with local users using SSHFS for home directories, and I am not finding anything similar.

**Comment by alkisg** [comment, marked as answer]
Hi, you can set UID_MIN and GID_MIN to 500 in /etc/login.defs. The pwmerge script reads them from there, as do other tools.
